packages/polaris-viz/src/playground/ExternalEvents.stories.tsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
+import type {Story} from '@storybook/react';
+import type {DataSeries} from '@shopify/polaris-viz-core';
+
+import {LineChart} from '../components/LineChart';
+import {PolarisVizProvider} from '../components/PolarisVizProvider';
+import {setSingleSeriesActive} from '../utilities';
+import {useWatchActiveSeries} from '../hooks';
+
+export default {
+  title: 'polaris-viz/Playground/ExternalEvents',
+  parameters: {},
+  decorators: [(Story) => <div>{Story()}</div>],
+  argTypes: {},
+};
+
+const DEFAULT_DATA: DataSeries[] = [
+  {
+    name: 'Breakfast',
+    data: [
+      {key: 'Monday', value: 3},
+      {key: 'Tuesday', value: -7},
+      {key: 'Wednesday', value: -7},
+      {key: 'Thursday', value: -8},
+      {key: 'Friday', value: 50},
+      {key: 'Saturday', value: 0},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+  {
+    name: 'Lunch',
+    data: [
+      {key: 'Monday', value: 4},
+      {key: 'Tuesday', value: 0},
+      {key: 'Wednesday', value: -10},
+      {key: 'Thursday', value: 15},
+      {key: 'Friday', value: 8},
+      {key: 'Saturday', value: 50},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+  {
+    name: 'Dinner',
+    data: [
+      {key: 'Monday', value: 7},
+      {key: 'Tuesday', value: 0},
+      {key: 'Wednesday', value: -15},
+      {key: 'Thursday', value: -12},
+      {key: 'Friday', value: 50},
+      {key: 'Saturday', value: 5},
+      {key: 'Sunday', value: 0.1},
+    ],
+  },
+];
+
+const CHART_ID = 'chart_01';
+
+const Template: Story = () => {
+  const [activeName, setActiveName] = React.useState('none');
+
+  useWatchActiveSeries(CHART_ID, ({detail: {index, data}}) => {
```

Comment:
There's an interesting bug that we're not understanding. When I hover over the button in Storybook, the event comes through with `detail: { index: 2 }` without a `data` key at all (not `undefined` or `null`).

I think we're not passing along `data` someplace we should be, which means even when you hover over the button, the text always says `Active series: none`

Reply:
I'm hesitant to pass the data back to consumers when triggering events outside of the chart. Ideally, they'd have the data they passed in anyways, so us passing the data prop back in events triggered from inside the chart is just a nice-to-have in case, for some reason, the data gets out of sync.

Reply:
@nickpresta What do you think about just removing `data` from the return until we find a need to get data back from the chart instead of using the data a consumer _should_ already have available?

Reply:
I think that's a fine assumption for now. We already assume (and require) the consumer to look at their own `data` object, anyways.